web/experimental/embed-new-flutter.html

```diff
@@ -129,7 +153,7 @@
 
 <footer flex layout horizontal class="footer">
     <div id="issues-message"></div>
-    <button id="issues-toggle" class="mdc-button mdc-button--dense mdc-button--raised" hidden></button>
+    <button id="issues-toggle" class="mdc-button mdc-button--dense mdc-button--outlined" hidden></button>
```

Comment:
The problem of keeping these three in sync raises its ugly head. 😆

Reply:
Yeah a templating system would be handy